### PR TITLE
Developer anand/registration update

### PR DESCRIPTION
**Pull request by developer-anand**
hey bro please accept my request.

Hey you are doing great work.

### CODE REVIEW

**Comment by gkanishk** [comment]
Hey @namancdr  ,
I just reviewed your **PR** and noticed you did everything the **wrong** way.
You have done changes is wrong  file you need to create a new directory with your username and put an **index.md** file with your details not edit the previous one.
This is how it would look like:
image
Kindly refer to the [Youtube Live](https://www.youtube.com/watch?v=QzjgBj9oaAA&t=850s) for the correct way.

**Comment by namancdr** [comment]
Bro i have already fixed this, & i am in first 50, why i am getting this
mail ?

**Comment by gkanishk** [comment]
Perfect @developer-anand it's good now start your takatakatakataka

File: content/participants/ttjc@developer-anand/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: registering
```
Comment (gkanishk) [inline content/participants/ttjc@developer-anand/index.md]: Kindly change the title with your name or username

**Comment by developer-anand** [comment]
I correct it sir
please accept it now.

**Pull request merged**